src/lib/utils/SynapseClient.ts

```diff
@@ -1114,6 +1114,11 @@ export const getPresignedUrlForWikiAttachment = (
     BackendDestinationEnum.REPO_ENDPOINT,
   )
 }
+
+export const isInSynapseExperimentalMode = ():boolean => {
```

Comment:
Also, add a way to ask if Synapse Experimental Mode is on (if the cookie exists).  Will use this to start to use the new Download List services.

Reply:
This is a nice addon, we can use this for other feature testing too.